src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.Com.cs

```diff
@@ -23,8 +23,7 @@ public static unsafe int AddRef(IntPtr pUnk)
             if (pUnk == IntPtr.Zero)
                 throw new ArgumentNullException(nameof(pUnk));
 
-            return CalliIntrinsics.StdCall__AddRef(((__com_IUnknown*)(void*)pUnk)->pVtable->
-                pfnAddRef, pUnk);
+            return ((delegate* unmanaged<IntPtr, int>)(*(*(void***)pUnk + 1 /* IUnknown.AddRef slot */)))(pUnk);
```

Comment:
Copy & paste from CoreCLR file

Reply:
Yes, I remember this line being popular on twitter.